src/test/java/VCSFileCreatorTest.java

```diff
@@ -0,0 +1,35 @@
+import org.example.gameService.VCSFileCreator;
+import org.example.model.GameData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+
+@ExtendWith(MockitoExtension.class)
+public class VCSFileCreatorTest {
```

Comment:
Please add more tests checking content/formatting of VCS file

Reply:
Done